virtualfish/virtual.fish

```diff
@@ -440,9 +442,20 @@ function __vf_help --description "Print VirtualFish usage information"
     echo
     echo "Available commands:"
     echo
-    for sc in (functions -a | sed -n '/__vf_/{s///g;p;}')
+
+    set -l subcommands (functions -a | sed -n '/__vf_/{s///g;p;}')
+    set -l max_subcommand_length 0
+    for sc in $subcommands
+        set -l cur (string length $sc)
+        if test $cur -ge $max_subcommand_length
+            set max_subcommand_length $cur
+        end
+    end
+    set -l spacing (math $max_subcommand_length + 2)
```

Comment:
> So in effect it will have no impact on the current commands

In my testing, this yielded three spaces between the longest subcommand (`globalpackages`) and the description column, whereas the gap was previously only two spaces. Not a big deal, obviously, but is there any harm in changing this line to: `set -l spacing (math $max_subcommand_length + 1)` ?